index.html

```diff
@@ -80,7 +83,9 @@ <h3 class="title-subheader">Aug 23-26  //  Portland, OR </h3>
 					</div>
 				</div>
 				<!-- ## COUNTDOWN TIMER END ## -->
-				<a id="slidelink" href="https://ti.to/write-speak-code/2017-conference/" target="_blank" class="qcTargetLink video cta register-cta ">Register Now</a>
+				<!-- <a id="slidelink" href="https://ti.to/write-speak-code/2017-conference/" target="_blank" class="qcTargetLink video cta register-cta ">
+				Register Now</a> -->
+				 <tito-button id="slidelink" event="write-speak-code/2017-conference" class="qcTargetLink video cta register-cta">Register Now</tito-button></li>						
 			</div>
```

Comment:
I would remove here `id=slidelink` and all the classes from the tito-button. The id is not necessary and those classes don't affect the button in the ways we want. Then I would just add the specific CSS in custom:
```css
.page-intro .tito-tickets-button {
	color: #fff;
	line-height: 60px;
	background: #EB6851;
	background-color: #EB6851;
	border: none;
	border-radius: 8px;
	letter-spacing: 1px;
	color: white;
	cursor: pointer;
	padding: 0 40px;
  margin-top: 50px;
	font-size: 16px;
}
.page-intro .tito-tickets-button:hover {
  background: rgba(0,0,0,0.3);
}
```